com.sayonara-player.sayonara.json

```diff
@@ -0,0 +1,149 @@
+{
+    "app-id": "com.sayonaraplayer.sayonara",
+    "branch": "master",
```

Comment:
```suggestion
```
You don't need to specify branch. And `master` isn't valid anyway.

Reply:
Removed in d1bb6c9